packages/manager/src/features/CloudPulse/Utils/FilterConfig.ts

```diff
@@ -0,0 +1,130 @@
+        filterType: 'string',
+        isFilterable: false,
+        isMetricsFilter: false,
+        name: 'Region',
+        neededInServicePage: false,
+        priority: 1,
+      },
+      name: 'Region',
+    },
+    {
+      configuration: {
+        dependency: ['region'],
+        filterKey: 'resource_id',
+        filterType: 'string',
+        isFilterable: true,
+        isMetricsFilter: true,
+        isMultiSelect: true,
+        name: 'Resource',
+        neededInServicePage: false,
+        placeholder: 'Select Resources',
+        priority: 2,
+      },
+      name: 'Resources',
+    },
+    {
+      configuration: {
+        filterKey: 'relative_time_duration',
+        filterType: 'string',
+        isFilterable: true,
+        isMetricsFilter: true,
+        isMultiSelect: false,
+        name: 'Time Duration',
+        neededInServicePage: true,
+        placeholder: 'Select Duration',
+        priority: 3,
+      },
+      name: 'Time Duration',
+    },
+  ],
+  serviceType: 'linode',
+};
+
+export const DBASS_CONFIG: CloudPulseServiceTypeFilterMap = {
+  filters: [
+    {
+      configuration: {
+        filterKey: 'dbEngine',
+        filterType: 'string',
+        isFilterable: false, // isFilterable -- this determines whethere you need to pass it metrics api
+        isMetricsFilter: false, // if it is false, it will go as a part of filter params, else global filter
+        isMultiSelect: false,
+        name: 'DB Engine',
+        neededInServicePage: false,
+        options: [
+          {
+            id: 'mysql',
+            label: 'MySQL',
+          },
+          {
+            id: 'postgresql',
+            label: 'PostgreSQL',
+          },
+        ],
+        placeholder: 'Select an Engine',
+        priority: 2,
+        type: CloudPulseSelectTypes.static,
+      },
+      name: 'DB Engine',
+    },
+    {
+      configuration: {
+        filterKey: 'region',
+        filterType: 'string',
+        isFilterable: false,
+        isMetricsFilter: false,
+        name: 'Region',
+        neededInServicePage: false,
+        priority: 1,
+      },
+      name: 'Region',
+    },
+    {
+      configuration: {
+        dependency: ['region', 'dbEngine'],
+        filterKey: 'resource_id',
+        filterType: 'string',
+        isFilterable: true,
+        isMetricsFilter: true,
+        isMultiSelect: true,
+        name: 'Resource',
+        neededInServicePage: false,
+        placeholder: 'Select DB Cluster Names',
+        priority: 3,
+      },
+      name: 'Resources',
+    },
+    {
+      configuration: {
+        filterKey: 'relative_time_duration',
+        filterType: 'string',
+        isFilterable: true,
+        isMetricsFilter: true,
+        isMultiSelect: false,
+        name: 'Time Duration',
+        neededInServicePage: true,
+        placeholder: 'Select Duration',
+        priority: 4,
+      },
+      name: 'Time Duration',
+    },
+  ],
+  serviceType: 'dbass',
```

Comment:
```suggestion
  serviceType: 'dbaas',
```
(See also the comment suggesting changing the type for `serviceType` to avoid these types of mistakes)

Reply:
changed the name

Reply:
changed